blog.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Adopt A Dog - Blog</title>
+  </head>
+
+  <body style="background-color:powderblue;">
+
+<!-- This is the header on the page - in two sections for styling later.  -->
+    <header>
+      <article id="left-header">
+        <a href="index.html"><img src="images/adoptadog-logo.svg" alt="Logo for Adopt A Dog with silouette of dog."></a>
+        <img src="images/header-image.jpg" alt="Background image of brown dog.">
```

Comment:
We'll cover later on in the course how to use these. For now, I'd just remove img all together and we'll come back to this in a few weeks.

Reply:
Do you mean to remove the svg images? Or ALL the images?

Reply:
Just the background image